### ISSUE TITLE
Add Cocoapod support

### ISSUE THREAD

**Issue by jlowgren** [slycrel/JSTileMap#35]
Perhaps something you've already considered? The [guide](https://guides.cocoapods.org/making/making-a-cocoapod.html) is pretty straight forward. It's nice to have all library dependencies handled by the same package manager.

**Comment by slycrel** [comment]
Yes, I have considered it.  Thanks for bringing it up again, maybe here in the next few weeks.  =)
